City-of-Pori/pori-kada
# PORI-344 News image issue fixes & PORI-359 Alt and Title missing from news image

To test:

1. drush fra -y && drush cc all
2. Go to any news that has a large image (such as https://local.pori.fi/uutinen/2018-02-23_paljasta-salaisuutesi-yyterin-salaisuudet-halutaan-julki) and make sure responsive image styles are used.
3. Make sure that on news image the title and alt values are also visible.

Not sure if this fixes the actual issue but at least the image sizes are smaller from now on.
